arquillian-chameleon-container-model/src/main/resources/chameleon/default/containers.yaml

```diff
@@ -147,7 +147,7 @@
   defaultProtocol: Servlet 3.0
   exclude: *EAP_EXCLUDE
 - name: JBoss AS
-  versionExpression: 7.*
+  versionExpression: 7\.[0-1]\.[0-2]\.(.*)$
```

Comment:
But this way we introduce a bug. We can have combined regex with more than one rule I believe - that would be one option for solving it.